core/storage/skill/gae_models.py

```diff
@@ -251,3 +252,45 @@ def get_export_policy():
     def get_user_id_migration_policy():
         """SkillSummaryModel doesn't have any field with user ID."""
         return base_models.USER_ID_MIGRATION_POLICY.NOT_APPLICABLE
+
+    @classmethod
+    def fetch_page(cls, page_size, cursor, sort_by):
+        """Returns the models according to values specified.
+
+        Args:
+            page_size: number. Number of skills to fetch
```

Comment:
"number" is not the right typeinfo.

Add period at the end.

Also just to check, the services layer should pass in a larger number than what the client desires, right? (due to taking filtering into account)

Reply:
Done.